phys/module_sf_noahmplsm.F

```diff
@@ -1831,7 +1861,8 @@ SUBROUTINE ENERGY (parameters,ICE    ,VEGTYP ,IST    ,NSNOW  ,NSOIL  , & !in
      IF(SNOWH.GT.0.)  THEN
          BDSNO    = SNEQV / SNOWH
          FMELT    = (BDSNO/100.)**parameters%MFSNO
-         FSNO     = TANH( SNOWH /(2.5* Z0 * FMELT))
+         !FSNO     = TANH( SNOWH /(2.5* Z0 * FMELT))
```

Comment:
Not required, as this is _your_ scheme. We find that removing old code is better than commenting it out. If this is left as an option for a developer, that is fine.

Reply:
The reason to keep the original code is that we want users/developers to know which parameter has been changed here, and as you mentioned developers can always swtich between these two.